### ISSUE TITLE
Place Edit / Preview switches in top bar

### ISSUE THREAD

**Issue by chromer030** [Automattic/simplenote-android#1196]
**What**
Place Edit / Preview switches in top bar

**Why**
This will free lots of room which can be very handy on smartphones which have small screens.

![Screenshot_20201111-195600_1](https://user-images.githubusercontent.com/13361552/98837783-9de9cf80-2458-11eb-93a6-39d819ae331b.png)

**Comment by theck13** [comment]
Thanks for the suggestion.  We don't want to add a toggling edit/preview action to the app bar for a few reasons.

Adding another action to the app bar would be too many.  There used to be too many and we moved all but two to the overflow menu.  Adding the edit/preview toggle to the overflow menu would require two taps to switch views.  We want switching between those views to be quick, which is why we chose tabs for easy swiping.  Additionally, enabling the views to be swiped allows the physical touch to be anywhere within the editor, which is roughly 80% of the screen.  That includes the lower part of the screen, which is easy for one-handed swiping with a thumb.  Moving the action to the top app bar or the overflow menu makes the action much more physically difficult to do with one hand, which is the typical use case for small form factor devices.

I have added it to our feature requests, but I don't see us changing this anytime soon due to the reasons above.
